client/js/chunks/cart.js

```diff
@@ -1,63 +1,45 @@
-const clearCart = document.querySelector('.clear-cart-btn');
-const openModalBtn = document.querySelector('[modal-open]');
-const closeModalBtn = document.querySelector('[modal-close]');
-const modal = document.querySelector('[modal]');
-const cartProdList = document.querySelector('.modal_product-list');
-const cartListHbs = require('../../../views/partials/cartList.hbs');
-const axios = require('axios');
-
-var Handlebars = require('hbsfy/runtime');
-const async = require('hbs/lib/async');
-Handlebars.registerHelper('priceCounter', (price, salePercentage) => {
+const clearCartBtn = document.querySelector(".clear-cart-btn");
+const openModalBtn = document.querySelector("[modal-open]");
+const closeModalBtn = document.querySelector("[modal-close]");
+const modal = document.querySelector("[modal]");
+const cartProductsList = document.querySelector(".modal_product-list");
+const buyBtns = document.querySelectorAll(".products_buy-btn");
+
+const cartListHbs = require("../../../views/partials/cartList.hbs");
+const { buyBtnHandler } = require("../buyBtnHandler.js");
+
+const axios = require("axios");
+axios.defaults.baseURL = "http://localhost:3000";
+
+var Handlebars = require("hbsfy/runtime");
```

Comment:
Why do you use var here?

It seems to be not suitable in your project. Just use let/const for your needs